README.md

```diff
@@ -18,35 +16,84 @@ This a python library used to transform data for data science libraries and pres
 [pypi-url]: https://pypi.python.org/pypi/RDT
 
 
+## Overview
+
+This a python library used to transform data for data science libraries and preserve the transformations in order to reverse them as needed.
+
+## Install
+
+### Requirements
+
+**RDT** has been developed and tested on [Python 3.5, 3.6 and 3.7][python-downloads]
+
+Also, although it is not strictly required, the usage of a [virtualenv][virtualenv] is highly recommended in order to avoid interfering with other software installed in the system where **RDT** is fun.
 
-## Installation
+These are the minimum commands needed to create a virtualenv using python3.6 for **RDT**:
+
+	pip install virtualenv
+	virtualenv -p $(which python3.6) rdt-venv
+
+Afterwards, you have to execute this command to have the virtualenv activated:
+
+	source rdt-venv/bin/activate
+	
+Remember about executing it every time you start a new console to work on **RDT**!
+
+[python-downloads]: https://www.python.org/downloads
```

Comment:
Let's use in-line links instead of variables in `md` documents. This is because the README ends up being parsed by m2r into the Sphinx docs and in some cases the variables have given us problems.